### PR TITLE
Add table headers to Summaries

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
This is a issue from the DAC report to make summaries more accessible by adding visually hidden table headers

### How to review 
See that table headers have been added and work well with screen readers

### CODE REVIEW

**Comment by rmccar** [comment]
[HIGH: Summary tables - hard to understand without table headers (GDS use <dl>) (pg.14 & 16) >> change summaries to definition lists](https://trello.com/c/ipTkykTl/715-high-summary-tables-hard-to-understand-without-table-headers-gds-use-dl-pg14-16-change-summaries-to-definition-lists)

File: src/components/summary/examples/hub/index.njk [hunk 1]
```diff
@@ -6,6 +6,7 @@
             {
                 "groups": [
                     {
+                    "headers":["Name of section or person", "Section progress", "Access section"],
```
Comment (boxadesign) [inline src/components/summary/examples/hub/index.njk:9]: Can runner provide just values and not keys? Maybe something to check.

Comment (rmccar) [reply src/components/summary/examples/hub/index.njk:9]: Should be able to I would think but ill check with the runner guys

Comment (rmccar) [reply src/components/summary/examples/hub/index.njk:9]: Spoken to the runner guys and they're happy with this implementation

**Review by boxadesign — APPROVED** [review]

**Pull request merged**